docs/source/daml-script/index.rst

```diff
@@ -366,13 +366,14 @@ To run DAML script against the JSON API you have to pass the ``--json-api`` para
    against a ledger that doesn't verify authorization. The section on
    :ref:`authorization <daml-script-auth>` describes how to specify
    the tokens.
-#. The tokens must contain exactly one party in ``actAs`` and/or
-   ``readAs``. This party will be used for ``submit`` and
-   ``query``. Passing a party as the argument to ``submit`` and
-   ``query`` that is different from the party in the token is an
-   error.
+#. The parties used for command submissions and queries must match the
+   parties specified in the token exactly.  For command submissions
+   that means ``actAs`` and ``readAs`` must match exactly what you
+   specified whereas for queries the union of ``actAs`` and ``readAs``
+   must match the parties specified in the query.
```

Comment:
The comment next to `validateTokenParties`, which is used for queries claims it checks for superset. Looks like that comment is off.